src/env.mjs

```diff
@@ -122,6 +122,17 @@ const server = z
     message: 'SENDGRID_FROM_ADDRESS is required when SENDGRID_API_KEY is set',
     path: ['SENDGRID_FROM_ADDRESS'],
   })
+  .refine(
+    (val) =>
+      val.NODE_ENV === 'development' ||
```

Comment:
I like to use unenv for things like this

Reply:
pass - following the pattern set down by previous authors